src/landscape/components/LandscapeView.js

```diff
@@ -116,19 +102,44 @@ const LandscapeView = () => {
   return (
     <PageContainer>
       <PageHeader header={landscape.name} />
-      <Typography
-        variant="caption"
-        display="block"
-        sx={{
-          marginBottom: theme.spacing(3),
-          marginTop: theme.spacing(2),
-        }}
-      >
-        {landscape.location}
-      </Typography>
+      <Restricted permission="landscape.change" resource={landscape}>
+        <Alert severity="info" sx={{ marginBottom: 2 }}>
+          <Trans i18nKey="landscape.view_manager_help">
+            {{ name: t('user.full_name', { user }), landscape: landscape.name }}
+            <Link href="TODO">link</Link>.
+          </Trans>
+        </Alert>
+      </Restricted>
       <Grid container spacing={2}>
         <Grid item xs={12} md={6}>
-          <LandscapeMap landscape={landscape} />
+          <Paper variant="outlined" sx={{ padding: 2 }}>
+            <Typography
+              variant="caption"
+              display="block"
+              sx={{
+                marginBottom: theme.spacing(2),
+              }}
+            >
+              {landscape.location}
+            </Typography>
+            <LandscapeMap
+              landscape={landscape}
+              label={t('landscape.view_map_title')}
+            />
+            <Restricted permission="landscape.change" resource={landscape}>
+              <Link component={Box} sx={{ marginTop: 2 }} href="">
```

Comment:
question: should href be blank?

Reply:
@paulschreiber check my answer to the previous comment